internal/replication/connect.go

```diff
@@ -50,3 +55,68 @@ func CheckPublication(db *sql.DB, name string) error {
 
 	return nil
 }
+
+func CreateSubscription(db *sql.DB, name string, connStr string) error {
+	sql := fmt.Sprintf(`CREATE SUBSCRIPTION %s CONNECTION %s PUBLICATION %s WITH (connect=false);`,
+		pq.QuoteIdentifier(name),
+		pq.QuoteLiteral(connStr),
+		pq.QuoteIdentifier(name))
+	_, err := db.Exec(sql)
+	if err != nil {
+		return err
+	}
+	log.Printf("subscription '%s' created", name)
```

Comment:
Ideally all log messages should be contextful, and in normal run ideally there should be at maximum one message per reconcile cycle.